### DISCUSSION TITLE
Feature request: disable SSID and password print-out through Serial

### DISCUSSION THREAD

**Discussion by i-g-g-y** [khoih-prog/ESP_WiFiManager_Lite#14] [Ideas]
Hello.

Is it possible to add a feature to disable the printout of SSID and password through serial port?

Great SW, happy to have found this.

**Comment by khoih-prog** [comment]
Just change `_ESP_WM_LITE_LOGLEVEL_` to 0. For example, from

https://github.com/khoih-prog/ESP_WiFiManager_Lite/blob/bc8fb7ed5159e78f4cba35e07bf8cb18a7925320/examples/ESP_WiFi/defines.h#L23

to 

```
#define _ESP_WM_LITE_LOGLEVEL_        0
```

**Comment by i-g-g-y** [comment]
OK, let me try this.

Thanks for fast response.

Great SW, really practical.

**Comment by i-g-g-y** [comment]
Yes, this works, thanks!
